src/components/WifiSetup/WifiList/WifiListEmpty.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import * as React from 'react';
+import { StyleSheet, Text } from 'react-native';
+import Container from '../../../common/Container';
+import { COLORS, TYPOGRAPHY } from '../../../theme';
+
+const styles = StyleSheet.create({
+  text: {
+    ...TYPOGRAPHY.secondary,
+    color: COLORS.textFaded,
+    paddingHorizontal: 12,
+    textAlign: 'center',
+  },
+});
+
+const WifiListEmpty = () => (
+  <Container centered>
+    <Text style={styles.text}>
+      There are not WiFi networks around the Brewskey box. Move your Brewskey
```

Comment:
We can't detect any nearby WiFi hotspots with your Brewseky box. Please move your Brewskey box and pull to refresh in the app.